mod/xlat/rfc7915.c

```diff
@@ -2782,7 +2784,7 @@ static void ttp64_icmp_err(struct xlation *state)
 	}
 
 	compute_icmp6_csum(out);
-	out->mark = IP6_REPLY_MARK(net, skb->mark);
+	out->mark = skb->mark; /* IP6_REPLY_MARK(net, skb->mark); */
```

Comment:
the IP6_* macros need CONFIG_IPV6. Just have to add that in menuconfig.

#ifdef'ing doesn't make sense as we need ipv6 anyway. We'll reflect this in Kconfig dependencies once we make the module in-tree.